tc/core/polyhedral/schedule_transforms.cc

```diff
@@ -128,8 +148,23 @@ isl::union_set activeDomainPointsHelper(
   }
   return domain;
 }
+
+// Get the set of domain elements that are active below
+// the given branch of nodes.
+isl::union_set activeDomainPointsHelper(
```

Comment:
Just change `activeDomainPoints` instead of introducing an extra one-line function?  Especially because it looks like you renamed the original `activeDomainPointsHelper`